src/components/DefaultLayout/DefaultLayout.js

```diff
@@ -11,28 +11,29 @@ const DefaultLayout = ({
   hideFooter,
   hideHeader,
   hideBrands,
-  hideShippingInfo, ...rest
-}) => {
-  return (
-    <Route {...rest} render={(matchProps) => (
+  hideShippingInfo
+}) => (
+  <Route
+    render={() => (
       <>
         {!hideHeader && <Header />}
-        <Component {...matchProps} />
+        <Component />
         {!hideBrands && <Brands />}
```

Comment:
I think it would be good to put all this page-parts into one component 
And render just 1 component for a route